sources/constraint-declaration-validation.asciidoc

```diff
@@ -640,14 +640,14 @@ The formal rules defining groups are as followed. _Text in italic are comments a
 For every class [classname]`X`:
 
 [start=1]
-. For each superclass [classname]`Y` of [classname]`X`, the group [classname]`Y` contains all constraints of the group [classname]`Y` of [classname]`Y`
+. For each superclass [classname]`Y` of [classname]`X`, the group [classname]`Y` contains all constraints of the group [classname]`Y`
```

Comment:
I'm not 100% sure about this one and would be prefer if we could keep that separately, discussing it with @emmanuelbernard. It may be meant in the sense of "group Y of [class] Y". I reckon it's incorrect indeed, but I'd rather have confirmation (in case Emmanuel remembers ;).